papas/aliceproto/PFfloodfill.py

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
Why is the name of this module PFfloodfill.py? 
shouldn't we change that to floodfill.py ? 
Please note that it's difficult for me to understand what are the differences w/r to the module I had provided, because you are still using the aliceproto directory. I might have more comments when you do a further PR to replace my code.